cinder/scheduler/filters/shard_filter.py

```diff
@@ -190,7 +191,11 @@ def backend_passes(self, backend_state, filter_properties):
                        'shard_prefix': self._SHARD_PREFIX})
             return False
 
-        if configured_shards_set & set(shards):
+        if self._ALL_SHARDS in shards:
```

Comment:
This is expected. But since `sharding_enabled` supersedes all other sharding tags, the shard tags become basically inert.

We will run a script over the projects to update with the new tag, and in all likelihood remove the old tags for these projects.